package.json

```diff
@@ -146,6 +146,7 @@
 		"rn-fetch-blob": "^0.12.0",
 		"rn-root-view": "RocketChat/rn-root-view",
 		"semver": "^7.3.8",
+		"transliteration": "^2.3.5",
```

Comment:
I'm pretty sure we added a slug lib a while ago

Reply:
This war the PR that introduced another type of "slug" https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/4364
That library works well with URI and parser. However, I was not able to use it as the library to slugify the non-ASCII character to ASCII. Also, I tested with other libraries, but the transliteration was the only one that was able to convert Russian, Arabic, Chinese, and Japanese characters to ASCII characters.